lib/views/login_register/forgot_password.dart

```diff
@@ -9,40 +10,25 @@ class ForgotPasswordScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      appBar: AppBar(backgroundColor: Colors.transparent, foregroundColor: Colors.black),
       body: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
-          const Padding(
-            padding: EdgeInsets.only(
-              top: 30,
-            ),
-            child: BackButton(
-              color: ThcColors.darkBlue,
-            ),
-          ),
```

Comment:
My preference is to use the `AppBar`'s built-in back button, so its size & position is consistent throughout the app.

Reply:
Great idea!